eclair-core/src/main/scala/fr/acinq/eclair/package.scala

```diff
@@ -44,6 +40,8 @@ package object eclair {
 
   def randomKey: PrivateKey = PrivateKey(randomBytes32)
 
+  def randomLong: Long = randomGen.nextLong()
```

Comment:
That's right, I get turned off because IntelliJ prompts me to remove the parenthesis, but they do make sense.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1774/commits/1e1037f13dc7744449dda518e0cb3fa0c280e6e6
I kept it in a separate commit that does only this, since it's trivial to review but touches a lot of test files (which is painful to read).